### ISSUE TITLE
Client port in UI

### ISSUE THREAD

**Issue by lars18th** [Barracuda09/SATPI#122]
Hi @Barracuda09 ,

I don't want to request for stupid and non-useful changes. However, some small changes in the UI can help _much_ when doing testing or debug (to the SATPI or to a SAT>IP client).

So, another missing information in the UI is the RTP port used by the client. In the Frontend Overview (and in the detailed view of the FE), the "Owner" is the address of the client. Great! But then can you please add the RTP port too? In fact the **target port**, as when using the RTSP protocol it will be the RTP (UDP base port), and when using the HTTP protocol is the TCP client port.

With this information will be more easy to know the port for example to launch a simple tcpdump.
You want to implement it, please?
Thank you!

**Comment by lars18th** [comment]
Hi @Barracuda09 ,

I hope you can implement this. It seems easy; and powerful for _testing_.
So, please can you add the TARGET PORT of the client in the UI?
Thank you!

**Comment by lars18th** [comment]
Hi @Barracuda09 ,

To be more precise: In the "Frontend Info", in the section "Stream Info", when you write the "Owner", the idea is to print not only the IP address of the client, but the pair of RTP UDP ports when streaming as well. Example: `Owner:    192.168.1.51:43002-43003`.

Thank you!

**Comment by lars18th** [comment]
Hi @Barracuda09 ,

I see the changes in the commit https://github.com/Barracuda09/SATPI/commit/cf539a4e4721f37f2d3f68cf71fe4b67b4c2bd03. However, after pull the last version and recompile, I don't see this in the web page. Only a new row "Stream URL" appears (even the link is wrong as it every time points to "http://satpi/frontend.html"). However, when I check for the "page source" the new lines for them are here:
```
page += addTableLineEntry("RTP Port", xmlDoc, streamID + "rtpPort");
page += addTableLineEntry("RTCP Port", xmlDoc, streamID + "rtcpPort");
page += addTableLineEntry("HTTP Port", xmlDoc, streamID + "httpPort");
```

You know what happens?

**Comment by Barracuda09** [comment]
Hi @lars18th 

- The "Stream URL" only is implemented for now, when using DVB-C/S2. So not for Child-PIPE/Streamer inputs.
- I am not sure why you don't see the lines, here is how I see it:

![image](https://user-images.githubusercontent.com/6542445/118373041-e4a9b500-b5b4-11eb-9b4e-42f8ca072e46.png)

**Comment by lars18th** [comment]
Yes, that's what I expected. However, in my case I only see the rows (but in the source of the page the code it's here):
- Stream URL
- Owner
- Session ID
- User-Agent
- RTP packet count
- RTP streamed (MB)

Please, remember that I've enabled ONLY virtual tuners: 
- Frontend 0: TS Reader
- Frontend 1: Streamer
- Frontend 2: Child PIPE - TS Reader
- Frontend 3: Child PIPE - TS Reader

Could this have any influence?

**Comment by lars18th** [comment]
My Chrome prints this in the console:
```
DevTools failed to load SourceMap: Could not load content for http://192.168.1.35:8080/assets/js/bootstrap.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for http://192.168.1.35:8080/assets/css/bootstrap.min.css.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
```
Any idea?

**Comment by Barracuda09** [comment]
The example picture I gave was with Child PIPE.

**Comment by Barracuda09** [comment]
> My Chrome prints this in the console:

I am not sure if this is the problem, as I have it as well

**Comment by Barracuda09** [comment]
Here you can get them and try it
https://github.com/twbs/bootstrap/releases/download/v4.5.3/bootstrap-4.5.3-dist.zip

**Comment by lars18th** [comment]
Hi @Barracuda09 ,

Problem fixed. **It's not related to the bootstrap**. However, I've updated it.

Before close this issue (now fixed/implemented) I request to enlarge a lot the default vertical size of the Frontend Info tab. Then we don't need to use the scroll as now just for a very small points the scroll is activated.

**Comment by Barracuda09** [comment]
@lars18th 

Yes, or maybe just have Bootstrap resize it (But I am not an expert in Web Development/Design I have managed to get this far, and with help from @issue )
